src/views/conviction-stop.njk

```diff
@@ -0,0 +1,72 @@
+{% extends "_base.njk" %}
+
+{% from "govuk/components/accordion/macro.njk" import govukAccordion %}
+{% block content %}
+  <h1 class="govuk-heading-xl">
+    You are not eligible to apply for a Fit and Competent licence
```

Comment:
Is this the correct term?
`a Fit and Competent licence`

I'm not sure, but I think we've used some different wording elsewhere?

Reply:
Resolved in 782b09e6848b484e5c7cfacb43ecdc4fd48280ee.